### DISCUSSION TITLE
Wixtooset under NixOS

### DISCUSSION THREAD

**Discussion by getreu** [wixtoolset/issues#8058] [Questions]
Does anyone have experience how to install and use the Wixtoolset under NixOS?

**Comment by chrpai** [comment]
The WiX Toolset only works on Windows. Nothing else is supported due deep dependencies on Win32 APIs namely the Windows Installer SDK.

**Reply by robmen** [reply]
Note: There *is* an issue open to support running on Linux, and some progress was made on it (not by the core team). Unfortunately, interest seemed to wane, and no one finished the work (I never saw any contributions). I actually changed the design of parts of WiX v4 to accommodate the work they described. 👻

**Reply by chrpai** [reply]
Yes, I know. I was there. 3000 years ago...  🤣

At this point a codespaces type service would be ideal but they are only investing in linux containers not windows containers.  I don't know if there are any competitors that would support this.

**Reply by chrpai** [reply]
A bunch of feedback over here might help. https://github.com/orgs/community/discussions/9977

**Comment by chrpai** [comment]
I see there is something called Microsoft Dev Box but I've never used it.

**Comment by getreu** [comment]
> The WiX Toolset only works on Windows. Nothing else is supported due deep dependencies on Win32 APIs namely the Windows Installer SDK.

I am confused. How is this working: [dactivllc/docker-wix: Docker container for creating Windows MSI packages using WiX Toolset](https://github.com/dactivllc/docker-wix) ?

Unfortunately the image is outdated. Is there an (official) alternative?
